### PR TITLE
Crypto-tracker 

### PR DESCRIPTION
**Pull request by Kandy-Peter**
**This PR, contains the following elements and requirements:**

- [ ] I added some interaction between two pages `home` and `details`.
- [ ] The data retrieved from the API should be stored in the Redux store.
- [ ] I have filtered the data that I retrieved from the API using a Filter stateless component.
- [ ] Every page (the main page and the pages for each item) have a unique route within the SPA.
- [ ] The project is deployed and accessible online (the link is in the README file).
- [ ] Create unit tests for pure functions in your Redux code.
- [ ] Create integration tests for your application using the React Testing Library

### CODE REVIEW

File: src/components/Cryptos.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+  const [search, setSearch] = useState('');
+
+  useEffect(() => {
+    dispatch(fecthCryptos());
+  }, []);
+
+  const searchStock = (e) => {
+    setSearch(e.target.value);
+  };
```
Comment (mikethreels) [inline src/components/Cryptos.js]: - [x] right now when searching, if the user makes a mistake and there is no such currency available the application gets stuck on the loading screen until the user reloads the screen, please fix this bug.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Kandy, 

Great job on this capstone project it looks great and very smooth, I'm impressed 👏 
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
- [ ] [OPTIONAL] it would be very nice if the user can search for the cryptocurrency by name as well instead of only by symbol.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Cryptos.js (hunk 1 above)
Comment (Kandy-Peter) [reply src/components/Cryptos.js]: Hi @mikethreels,
Thanks for your feedback. I'm going to solve it but for the optional feedback, It's not possible to do that due to the API that doesn't provide the crypto name. It only gives symbols and other details.

**Review by elyor-doniyorov — APPROVED** [review]
Hi @Kandy-Peter, 

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**